lib/locales/index.ts

```diff
@@ -15,7 +15,7 @@ import { locale as Sk } from './sk';
  * An BCP-47 IETF key to value mapping of all supported locale resource files
  * in romcal.
  */
-export const locales: Record<Lowercase<string>, Locale> = {
```

Comment:
Did you do make this change in order to fix #389? If yes, did you fix all the TS issues? If yes, consider adding `fix #389` to the PR description.

Reply:
I hadn't seen this issue. But yes indeed, this PR fixes all TS typing errors, and will close https://github.com/romcal/romcal/issues/389.

Reply:
I think you should consider watching _All Activity_ in this repository (as I do), so that you don’t miss a notification about any new issues/discussion/comment. I don’t always mention you. :wink:

Reply:
This is already the case. But I have a very busy schedule, and I don't have time to go into all the subjects in depth and exchange on it (this is why I sometimes miss out certain points). 

At least, I try to skim though updates to keep an eye on the project. But I recognize that I rely a little on you 😄, because I know that you have more the issue tracker in mind than me.